### PR TITLE
added search Ingredients by phrase using RequestParam

### PR DESCRIPTION
**Pull request by MajewskiOliwier**

### CODE REVIEW

File: src/main/java/com/example/JavaProject/controller/IngredientController.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+public class IngredientController {
```
Comment (Karol2846) [inline src/main/java/com/example/JavaProject/controller/IngredientController.java:15]: RecipeService and AuthenticationService are not used, you should remove them

and optimise imports for this file

File: src/main/java/com/example/JavaProject/repository/IngredientRepository.java [hunk 2]
```diff
@@ -11,5 +11,6 @@ public interface IngredientRepository extends JpaRepository<Ingredient, Long> {
     Ingredient findByIngredientNameAndMeasurement(String ingredient, String measurement);
     Boolean existsById(long id);
```
Comment (Karol2846) [inline src/main/java/com/example/JavaProject/repository/IngredientRepository.java]: this method is not used, you can remove it
Edit. i can edit your comment ... weird

**Review by Karol2846 — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/JavaProject/repository/IngredientRepository.java (hunk 2 above)
Comment (MajewskiOliwier) [reply src/main/java/com/example/JavaProject/repository/IngredientRepository.java]: Check out newest commit

**Review by Karol2846 — APPROVED** [review]
Okey, now everything is fine :)

**Pull request merged**